sdk/storage/azure-storage-files-datalake/src/datalake_path_client.cpp

```diff
@@ -274,7 +274,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Models::CreateDataLakePathResult ret;
     ret.ETag = std::move(result->ETag.GetValue());
     ret.LastModified = std::move(result->LastModified.GetValue());
-    ret.ContentLength = std::move(result->ContentLength);
+    ret.FileSize = std::move(result->ContentLength);
```

Comment:
It's meaningless to `std::move` from an integer. Actually I remember some compiler or code static analysis tool will give a warning.

Reply:
It is a nullable object.